coord/src/dist_aggregate.c

```diff
@@ -220,6 +235,18 @@ static const RLookupKey *keyForField(RPNet *nc, const char *s) {
   return NULL;
 }
 
+void processResultFormat(uint32_t *flags, MRReply *map) {
```

Comment:
Yes, but as discussed, will require obtaining a ref to the index spec (need to detect whether index is on hash or json)

Reply:
We do want to obtain a ref to the index spec from the background since it will require locking the GIL (confirmed with @GuyAv46)
So we cannot process the FORMAT arg in the coordinator and we will keep getting the format from shards.